consensus/vbft/service.go

```diff
@@ -1058,6 +1058,31 @@ func (self *Server) processProposalMsg(msg *blockProposalMsg) {
 		log.Errorf("BlockPrposalMessage check blocknum:%d,prevhash:%s,msg prevhash:%s", msg.GetBlockNum(), hex.EncodeToString(prevBlkHash[:4]), hex.EncodeToString(msgPrevBlkHash[:4]))
 		return
 	}
+	cfg := vconfig.ChainConfig{}
+	if blk.getNewChainConfig() != nil {
+		cfg = *blk.getNewChainConfig()
+	} else {
+		if blk.getLastConfigBlockNum() != math.MaxUint32 {
```

Comment:
If there's no new config in proposal, just check if the new proposal should has new config?